compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -671,6 +671,9 @@ pub enum TerminatorKind<'tcx> {
         /// This allows the memory occupied by "by-value" arguments to be
         /// reused across function calls without duplicating the contents.
         args: Vec<Operand<'tcx>>,
+        /// The spans of the args
+        /// (e.g. `a` and `b` in `x.foo(a, b)`).
```

Comment:
```suggestion
        /// (e.g. `a` and `b` in `x.foo(a, b)`).
        /// Has the same length as `args`.
```

Reply:
I agree this would have been good to add, but now that I started using `Spanned<Operand<'tcx>>` there is nothing that can get out of sync any longer.